mpisppy/__init__.py

```diff
@@ -1,18 +1,26 @@
 # Copyright 2020 by B. Knueven, D. Mildebrath, C. Muir, J-P Watson, and D.L. Woodruff
 # This software is distributed under the 3-clause BSD License.
-import os
-from pyutilib.misc.timing import TicTocTimer
+
+import pyomo as _pyo
+
+pyomo6 = (int(_pyo.__version__[0]) >= 6)
+
+if pyomo6:
+    from pyomo.common.timing import TicTocTimer as _TTT
+else:
+    from pyutilib.misc.timing import TicTocTimer as _TTT
+
 try:
-    import mpi4py.MPI as mpi
+    import mpi4py.MPI as _mpi
```

Comment:
Not particularly important but any reason we prepended `mpi` with an underscore?

Reply:
Only in case somebody does `from mpisppy import *`. The names with leading underscores will *not* be imported (and they're local here anyhow).